python/lsst/skymap/detail/dodecahedron.py

```diff
@@ -10,9 +12,12 @@ class Dodecahedron:
     """
 
     def __init__(self, withFacesOnPoles=False):
-        """Construct a Dodecahedron
+        """Construct a Dodecahedron_
```

Comment:
extra `_`?

Reply:
Missed that, I was trying to make package-docs complain to confirm it was seeing dodecahedron.py